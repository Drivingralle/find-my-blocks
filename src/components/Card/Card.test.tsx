import React from "react";
import renderer from "react-test-renderer";
import { Card } from ".";

it("renders the Content", () => {
  const tree = renderer.create(<Card>Hello</Card>).toJSON;
  expect(tree).toMatchSnapshot();
});
test("adds 1 + 2 to equal 3", () => {
  expect(true).toBe(true);
});
