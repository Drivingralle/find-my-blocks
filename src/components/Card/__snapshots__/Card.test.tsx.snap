// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card component should renders correctly 1`] = `
<div
  className="card"
>
  <div
    className="title"
    onClick={[Function]}
  >
    Toronto
  </div>
  <div
    className="body"
  >
    Go Raps!
  </div>
</div>
`;
